cirq-superstaq/cirq_superstaq/job.py

```diff
@@ -225,3 +235,20 @@ def __repr__(self) -> str:
 
     def _value_equality_values_(self) -> Tuple[str, Dict[str, Any]]:
         return self._job_id, self._job
+
+
+def _get_marginal_counts(counts: Dict[str, int], indices: Sequence[int]) -> Dict[str, int]:
+    """Compute a marginal distribution, accumulating total counts on specific bits (by index).
+
+    Args:
+        counts: The dictionary containing all the counts.
+        indices: The indices of the bits on which to accumulate counts.
+
+    Returns:
+        A dictionary of counts on the target indices.
+    """
+    target_counts: Dict[str, int] = collections.defaultdict(int)
+    for bitstring, count in counts.items():
+        target_key = "".join([bitstring[index] for index in indices])
+        target_counts[target_key] += count
+    return target_counts
```

Comment:
```suggestion
    return dict(target_counts)
```
(so that the returned value is not a defaultdict)